en/deploy-br-federation-manager.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Deploy BR Federation Manager on Kubernetes
+summary: Learn how to deploy BR Federation Manager on Kubernetes.
+aliases: ['/docs/tidb-in-kubernetes/dev/deploy-br-federation-manager/']
+---
+
+# Deploy BR Federation Manager on Kubernetes
+
+This document describes how to deploy BR Federation Manager across multiple Kubernetes.
+
+## Prerequisites
+
+Before deploy BR Federation Manager on Kubernetes, you should satisfy conditions below:
+
+* Kubernetes >= v1.12
+* you have multiple Kubernetes clusters
+* All the Kubernetes clusters that serve as data planes have deployed TiDB-Operator
+
+## Deploy BR Federation Manager
+
+### Generate Kubeconfig file in all Data Planes
+
+BR federation manager manages Kubernetes clusters of data planes by accessing their API servers.
+So it needs kubeconfig to authenticate and authorize in the API servers. The users or service accounts in the kubeconfig file need
+have all the permissions of **backups.pingcap.com, restores.pingcap.com** CRD at least.
+
+You can get the kubeconfig file from the Kubernetes cluster administrator. But we also provide a way to generate
+the kubeconfig file by yourself if you have the permission to access all the data planes.
+
+#### Create Resources about RBAC in Data Plane
```

Comment:
```suggestion
### Step 1.1: Create Resources about RBAC in Data Plane
```

Reply:
Numbering the steps will make it clearer for users to navigate through the doc. Can you please update all the other headings in this format?